src/common/components/hosts/utils.ts

```diff
@@ -120,13 +120,19 @@ export const canHostnameBeChanged = (hostStatus: Host['status']) =>
     'pending-for-input',
   ].includes(hostStatus);
 
-export const getHostRole = (host: Host, t: TFunction, schedulableMasters?: boolean): string => {
+export const getHostRole = (
```

Comment:
the function is called from `StorageUtils.tsx` which is not passing the new `clusterKind` prop